Penteract/Assets/Prefabs/SpriteSheetExperiment

```diff
@@ -0,0 +1,818 @@
+{
```

Comment:
This prefab doesn't have the .prefab extension, if you're needing it, add the extension, otherwise it won't exist in the Assets/Prefabs

Reply:
Prefab removed